src/main/java/com/minecolonies/coremod/blocks/ModBlocks.java

```diff
@@ -35,6 +35,8 @@
     public static BlockMinecoloniesRack       blockRack;
     public static BlockTimberFrame            blockTimberFrame;
     public static BlockWaypoint               blockWayPoint;
+    public static BlockHutBarracks            blockHutBarracks;
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Make this "public static blockHutBarracks" field final [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3AS1444)
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Make blockHutBarracks a static final constant or non-public and provide accessors if needed. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3AClassVariableVisibilityCheck)